src/honeybadger.cr

```diff
@@ -30,4 +30,14 @@ module Honeybadger
 
     Dispatch.send payload, synchronous: synchronous
   end
+
+  private EVENT_DISPATCH = EventDispatch.new
+
+  def self.event(**properties)
+    send Event.new(**properties)
+  end
```

Comment:
I think this will need to merge in some of the defaults, at least to capture the environment.

Reply:
Good catch

Reply:
I'm surprised by this, but the client spec actually says not to do that -- to leave the event as bare as possible:

> We want to keep “event” payloads open, as this is the base for all other kinds of events (either from Honeybadger or custom user events) with just event_type  and ts  as required keys. All other supplied payload data is merged to the root of the event.

Reply:
I think it's ok if you want to add a few default fields like environment, assuming they can be overridden by the caller. We ended up adding a before_event hook to the Ruby client specifically to enable adding additional defaults like environment. :) We may yet add that to the default payload.

Reply:
Would it make sense to defer the defaults to a followup PR if other clients aren't using them (yet)?